src/components/VueScriptComponent.vue

```diff
@@ -1,44 +1,48 @@
 <template>
-  <div :id="compId">
-  </div>
+  <div :id="compId" :key="id"></div>
 </template>
 
 <script>
-import postscribe from "postscribe"
-var ready = require('document-ready')
+import postscribe from "postscribe";
+var ready = require("document-ready");
 
 export default {
   name: "VueScriptComponent",
-  props:["script"],
   data() {
+    let randomID = Math.random()
+      .toString(36)
+      .substring(7);
     return {
-      compId: Date.now() //we will use date.now as a soft ID so that users may load multiple vue-script-components without conflicting ids
+      compId: window.performance.now() + "-" + randomID
     };
   },
-
+  props: {
+    id: {
```

Comment:
this prop is always null?

Reply:
The `id` prop has been removed in the latest commit.